File: Selenium/Activities/Activity16.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.select import Select

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/selects")
    print(f"Title of the page: {driver.title}")

    drop_down = driver.find_element(By.XPATH, "//select[contains(@class, 'h-10')]")
		
    select = Select(drop_down)
		
    select.select_by_visible_text("Two")
    select.select_by_index(3)
    select.select_by_value("four")
		
    options = select.options
    print("Options : ")
    for option in options:
        print(option.text)


        

File: Selenium/Activities/Activity16.java
import java.util.List;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.Select;

public class Activity16 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		
		driver.get("https://training-support.net/webelements/selects");
		System.out.println("Title of the page: " +driver.getTitle());
		
		WebElement dropDown = driver.findElement(By.xpath("//select[contains(@class, 'h-10')]"));
		
		Select select = new Select(dropDown);
		
		select.selectByVisibleText("Two");
		select.selectByIndex(3);
		select.selectByValue("four");
		
		List<WebElement> options = select.getOptions();
        System.out.println("Options : ");
        for(WebElement option : options) {
            System.out.print(option.getText() + " ");
        }
        
        driver.quit();
	}

}
